pyfolio/tests/test_perf_attrib.py

```diff
@@ -447,3 +451,18 @@ def test_high_turnover_warning(self):
             "This algorithm has relatively high turnover of its positions.",
             str(w[-1].message),
         )
+
+    def test_cumulative_returns_less_costs(self):
+
+        returns = generate_toy_risk_model_output()[0]
```

Comment:
What would you think of hardcoding `returns` to a small array (like we do with `cost`, and then hand deriving the expected values below? Otherwise we're essentially just reimplementing the logic of the function in this test

Reply:
true, hard coded the expected values